src/librustdoc/html/static/css/settings.css

```diff
@@ -6,41 +6,44 @@
 .setting-line > div {
 	display: inline-block;
 	vertical-align: top;
-	font-size: 17px;
+	font-size: 1rem;
 	padding-top: 2px;
 }
 
-.setting-line > .title {
-	font-size: 19px;
-	width: 100%;
-	max-width: none;
-	border-bottom: 1px solid;
-}
-
-.setting-line .radio-line,
 .setting-line .choices {
 	display: flex;
 	flex-wrap: wrap;
 }
 
-.setting-line .radio-line .setting-name {
-	flex-grow: 1;
-	margin-top: auto;
-	margin-bottom: auto;
-}
-
 .setting-line .radio-line input {
 	margin-right: 0.3em;
+	height: 1.2rem;
+	width: 1.2rem;
+	border: 1px solid;
+	outline: none;
+	-webkit-appearance: none;
+	cursor: pointer;
+	border-radius: 50%;
+}
+.setting-line .radio-line input + span {
+	padding-bottom: 1px;
+}
+
+.radio-line .setting-name {
+	width: 100%;
 }
 
 .radio-line .choice {
-	border-radius: 0.1em;
-	border: 1px solid;
-	margin-left: 0.5em;
 	margin-top: 0.1em;
 	margin-bottom: 0.1em;
 	min-width: 3.8em;
 	padding: 0.3em;
+	display: flex;
+	align-items: center;
+	cursor: pointer;
```

Comment:
I think `pointer` is not needed for `.choice`. It is the default style for labels. It's only needed for the `input` ([for some reason radio buttons don't have pointer by default](https://ux.stackexchange.com/questions/23266/should-the-cursor-change-to-pointer-over-radio-buttons-checkboxes-their-labels)).

Reply:
No, apparently it's needed.